examples/ibis/models/ibis_full_model.py

```diff
@@ -0,0 +1,25 @@
+import ibis
+
+from sqlmesh.core.model import model
+from sqlmesh.core.macros import MacroEvaluator
+
+@model(
+    "ibis.ibis_full_model",
+    is_sql=True,
+    kind="FULL",
+    audits=["assert_positive_order_ids"]
+)
+def entrypoint(evaluator: MacroEvaluator) -> str:
+    # connect to database
+    con = ibis.duckdb.connect(database="data/local.duckdb")
```

Comment:
@tobymao is it possible to get the database from `MacroEvaluator`? If not, maybe use an environment variable here and in config.py, so we don't have to duplicate the database file location.

Reply:
the macro evaluator has a reference to engine_adapter

Reply:
i'd just stick with a constant at that point.

Reply:
hm wait a second, this is a python sql model and not a python model. is it possible to have one example for both? in this case, why do you need any connection to the database, you just need to generate sql?